torch/_dynamo/decorators.py

```diff
@@ -159,6 +165,166 @@ def forbid_in_graph(fn):
     return fn
 
 
+def substitute_in_graph(
```

Comment:
Wasn't this already added in #133712

Reply:
I think this is a bug for `ghstack`. I'll try to figure it out.